### ISSUE TITLE
add an example project

### ISSUE THREAD

**Issue by aaronpeikert** [aaronpeikert/repro-tutorial#226]

**Comment by aaronpeikert** [comment]
@brandmaier @cjvanlissa I think this a wontfix. The invest/reward ratio is off. This will be a lot of work, but repro already contains an example, the PAC is an example and the paper itself is an example. What do you think?

**Comment by brandmaier** [comment]
Yes, point to the existing example - that is enough! Reviewers may simply not have been aware of these existing resources.
